src/logic/localStorage.js

```diff
@@ -0,0 +1,39 @@
+
+export const getStoredData = () => {
+  const storedData = localStorage.getItem("storedData");
+  return storedData ? JSON.parse(storedData) : [];
+};
+
+export const setStoredData = (data) => {
+  localStorage.setItem("storedData", JSON.stringify(data));
+};
+
+export const addData = (type, text) => {
+  const storedData = getStoredData();
+  const newItem = { type, text, timestamp: new Date().toISOString() };
+  storedData.push(newItem);
+  setStoredData(storedData);
+};
+//Zostawiam to na wypadek dodania funkcji uswania chatu 
```

Comment:
Stick to english comments.

Reply:
Also no need for it to be commented. As long as you won't invoke it this won't do anything so it can stay as it is.